core/client/src/main/java/alluxio/client/block/BlockWorkerInfo.java

```diff
@@ -56,6 +56,13 @@ public long getCapacityBytes() {
   }
 
   /**
+   * @return the available bytes of the worker
+   */
+  public long getAvailableBytes() {
```

Comment:
I think you can use a fixed number based on some factor * the default block size. The factor will be determined by the number of concurrent blocks you expect to be written to a worker at once. I suggest to make this configurable.